packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed 
+* Fixed Page Scroll Issue of form-select.
```

Comment:
Can these entries be consolidated? Each item in a list should be for a specific change, here there are three asterisks for one entry.

Example:

### Fixed 
* Fixed page scrolling when embedded within an iframe

Reply:
sure, updated [change-log](https://github.com/cerner/terra-core/pull/2773/commits/a428f6c4bd960d219d453cc40d5239830ecb81cd)